tests/test_perf_tracking.py

```diff
@@ -2227,26 +2227,6 @@ def test_empty_positions(self):
             self.assertEquals(val, 0)
             self.assertNotIsInstance(val, (bool, np.bool_))
 
-    def test_update_last_sale(self):
```

Comment:
If Position objects still hold last sale prices, do we want to test for updating that field?